### DISCUSSION TITLE
[Paid job | Google Play | PlayAssetDelivery.ane] Shapik: The Moon Quest game

### DISCUSSION THREAD

**Discussion by paulp-games** [airsdk/Adobe-Runtime-Support#2186] [General]
Hello,

the size of my game is 760mb. Before I used APK + OBB(distriqt.ExpansionFiles.ane), but now AAB doesn't support extension files. Need to add support for [PlayAssetDelivery](https://github.com/airsdk/ANE-PlayAssetDelivery)

Google Play: https://play.google.com/store/apps/details?id=air.shapikthemoonquest

Game build in Flash Develop(starling 1.8 + air 33.1)

**Comment by balaedit** [comment]
If you need some help regarding packaging, please mail details to balaedit1987@gmail.com

**Comment by caraiz** [comment]
Hello, if you still searching, my email is carai2@hotmail.com, thanks.
